Translations for fr v1.8.0

JaseurPattu opened the pull request.

nnnko56 reviewed (approved):
**General Opinion**
After reviewing all the submitted files of this PR,  I recommend it to be merged in the code.

There are some errors and corrections needed and there's place for improvement in multiple instances,  but the volume of work in this PR is massive and canot be dismissed. In addition, it will be a lot simpler to make smaller changes afterwards.

**Review Format:**
With my review there is close to 300 line comments with fixes, suggestions, or remarks. The vast majority are minor edits (punctuation, grammar, typos)  and could be tackled later on in a follow-up PR.

All my comments and suggestions can be tracked with this simple pseudo-code format in the comments:

[new word]  <=  suggested added word or letter
{removed word}  <= suggested for removal 
< old | new >  <= suggested for replacement

For example :

This is a [bigger] sentence     =>  This is a bigger sentence
This is a {paragraph} sentence  =>  This is a sentence
This is < a big | an adequate > sentence. =>  This is an adequate sentence

**Before Next Release:**
There is however some errors that I feel should at least be corrected before the next release.  For clarity I will relist them here, and you can refer to the corresponding line comment for details and a suitable replacement if needed:



app/i18n/docs/fr/Tutorials/DecredIntoPage02.md
- The translation says that PoS can "replace" PoW, which is not accurate. PoS can Override PoW Blocks but PoS cannot create blocks.


app/i18n/docs/fr/Tutorials/DecredIntoPage01.md  and more (search comments and pr for "Jeton")
- Decred is refered to as Tokens instead of money. In most occasion "jeton" ou "jetons" , should be replaced by "monnaie" see line comments


app/i18n/translations/fr.json

1- "export.transactions.fields.credits" :
The proposed translation of "transaction outputs" would means "outgoing transactions",  which for addresses in a wallet technically is the opposite of what is intended.

2- "export.transactions.fields.debits"
The proposed translation of "transaction inputs" would means "incoming transactions",  which for addresses in a wallet technically is the opposite of what is intended

3- "mixer.start.running":
is a translation error, could be a copy/paste error but it says to stop the mixer in order to start the mixer. ( should be: in order to buy a ticket)



**Standardization of terms:**
Outside of the these errors there are certain words that reoccur often in our texts and that should be standardized to make things easier to read and to use industry recognized terms. In some case it's extremely hard because there is "no" good translation, but we should at least attempt to make our docs consistent with how we use them. Ideally these should be discussed before the next PR. Somme of these are tagged with line comments and some are not. Also, only submitted edits from the pr have been reviewed and not the unchanged content of these files.  Instances can be found by searching the words.

**PdT/PoW  PdE/PoS**
These acronyms and the corresponding terms are used multiple times in the translation. But the format vary a lot. sometimes the English acronym is in parenthesis , or there is a  /,  or it's not there or Only the English acronym is used.  Ideally we should always write it the same way.
Maybe : 
- PdT (PoW)
- PdE (PoS)
We could reserve the / for the hybrid nature of Decred  PdT/PdE
And should we keep the acronym along when using the longer name ?
- Preuve de Travail (PdT)
-  Preuve d'Enjeu (PdE)


**Stake:**

Stake is used as-is without translation in multiple instances but we do have an accurate translation for Proof-Of-Stake (Preuve d'enjeu).  In order to be consistent, the stake could be something like "l'enjeu"  And we could decide to always include the English word for clarity :   - Enjeu (stake)



**staking**
Staking is the worst of all. There is no "good" translation for the action of putting in stake in French except maybe "parier" which means gamble and is inaccurate in this context. I would suggest between the following:

- Participation à la preuve d'enjeu (staking)
 - Mise en enjeu (staking)
 - Enjeu (staking)
 - Participation à l'enjeu (staking)


**Stakeholders**
Similar to the previous ones, maybe :

- Détendeurs d'enjeu
- Participants à l'enjeu
- Participant à la preuve d'enjeu




**automatisé**
In almost all occurrences, I feel like "automatique" should be used



**périphérique**
For Trezor and other hardware wallets, the more accurate term would be "dispositif". Because "Périphérique"/peripheral is usually a device that extends the functions of a computer and offer no function by itself.


**paramétrage**
The usual words for software settings in French is "Configuration"  (general),   paramètre ( one setting), paramètres (a group of settings)


**sécurisé**
Sécurisé is not the best choice of word. It tries to replicate the English word "secure" but means something that has been made secure (was not previously). In french we can simply say "sûr" or "sécuritaire" which means it simply "IS" secure.

**Additional remarks**
I have noticed a few entries in the fr.json files that have no corresponding entries in the source English file. Unused entries have no impact on the user experience, but a little clean-up should be made to remove unused entries.


Finally I'm sure it is known already but the English source needs updating after  1/89/10.  For now translation files "correctly" mirror the outdated values from the source English file.

xaur commented:
Thanks for your epic translation and review @JaseurPattu and @nnnko56 !

@nnnko56 do I understand correctly that the most urgent fixes we need to make this translation release-ready are the 5 items listed in the Before Next Release section of your [comment](https://github.com/decred/decrediton/pull/3895#pullrequestreview-1647819254), while everything else can be polished in future PRs?

Totally agree about standardization of terms. We will save time if we do it once and document non-trivial terms, recommended translations, and any reasoning or comments for them. We can put it all in [this French document](https://github.com/decredcommunity/translations/blob/master/docs/french.md) before doing the next polishing PR.

nnnko56 commented:
> Thanks for your epic translation and review @JaseurPattu and @nnnko56 !
> 
> @nnnko56 do I understand correctly that the most urgent fixes we need to make this translation release-ready are the 5 items listed in the Before Next Release section of your [comment](https://github.com/decred/decrediton/pull/3895#pullrequestreview-1647819254), while everything else can be polished in future PRs?
> 
> 

Yes the 5 items are the ones I recommend  be fixed before the next release because the meaning is not accurate and could lead to confusion.  Of the 5 ,  4 are sentence replacement and I provided alternatives I believe are suitable.

The other issue is the use of the word "jeton"  (token). I know it's often a a sensitive subject in chats because Decred is a currency (Money) with it's own blockchain and not a token which often refers to (ETH tokens).  So we may want to not call Decred a token (jeton) in our official docs,  but it's more work because it means replacing multiple lines. I tried to make line comments and review each occurrence but I may have missed some and there may be other occurrences outside of this PR so so it may or may not be as urgent to fix  depending on the required work and the time available.,

JaseurPattu commented:
First of all, many thanks for the review @nnnko56, I'm really happy to see this PR moving forward !

I agree with both of you and think it is wise to focus on the 5 keys things you have identified.

As referred by @xaur, I started a standardization page to describe keys translation decisions and this page has determined the translation of some blockchain/technical terms in this work.
Following you review, I will update it and keep it in sync to reflect latest decisions.

About the "coin" translation:
To be honest, it was complicated to me to decide between the terms "monnaie", "jeton" or even "pièce". 
Back then, I have chosen to follow the french legal/fiscal point of view where cryptocurrencies are not recognized as a official money but as tokens (same has collector gold coins in fact).
I will of course update to "monnaie" as you requested but at least you have now the rationale for the decision !

Please note that a lot of labels are not in english translation file and are specified only in UI screens files. 
While testing Decrediton and translation, I have added intentionally these translations in order to avoid having a mix of english/french on screens. So, it is important to keep them to have the best user experience.
Unfortunately, I had no time to add them in english translation file but that would be an effort to do ideally as it would be beneficial for other languages too.

For other comments, I have overall no objections on most of them, but I will comment back if I need some clarification or to discuss your proposal.

I agree that "stak*" terms are the worse to translate, and moreover they sound bad in french !
By chance, would you be available to chat about them one day ?

xaur commented:
> a lot of labels are not in english translation file and are specified only in UI screens files. While testing Decrediton and translation, I have added intentionally these translations in order to avoid having a mix of english/french on screens. So, it is important to keep them to have the best user experience.

@JaseurPattu which strings do you refer to? Do you mean you've added French strings to `app/i18n/translations/fr.json` which are not currently used because they are hardcoded in UI code?

xaur commented:
> Unfortunately, I had no time to add them in english translation file but that would be an effort to do ideally as it would be beneficial for other languages too.

Right, but let's leave that for other PRs. We want this PR to have minimum conflicts to be easier to merge.

JaseurPattu commented:
> > a lot of labels are not in english translation file and are specified only in UI screens files. While testing Decrediton and translation, I have added intentionally these translations in order to avoid having a mix of english/french on screens. So, it is important to keep them to have the best user experience.
> 
> @JaseurPattu which strings do you refer to? Do you mean you've added French strings to `app/i18n/translations/fr.json` which are not currently used because they are hardcoded in UI code?

What I was referring to is that some entries in the UI code are missing in the english translation.
I have added them as entries in the french translation to have them in french (no UI code change required, it works out of the box)

**For instance:**
Label with key: "confDialog.signTransaction.message"
Is defined in UI code here along with its default english text: https://github.com/decred/decrediton/blob/master/app/wallet/confirmationDialog/signTx.js#L24
But cannot be found in original.json: https://github.com/decred/decrediton/blob/master/app/i18n/translations/original.json

JaseurPattu commented:
> > Unfortunately, I had no time to add them in english translation file but that would be an effort to do ideally as it would be beneficial for other languages too.
> 
> Right, but let's leave that for other PRs. We want this PR to have minimum conflicts to be easier to merge.

Yes, it is clearly out of scope of this PR

xaur commented:
v1.8.1 is out but we may have a v1.8.2 with patches for Ledger, we might jump into that. @JaseurPattu please check the chat when you get a chance.

JaseurPattu commented on app/i18n/docs/fr/Tutorials/IdentityPage03.md:
```diff
@@ -0,0 +1 @@
+In CMS, identity is already used to sign invoices and comments, and possibly DCCs and DCC votes (not sure, need to check). In the future, I hope it will be also used to sign proposal owner's approval of billing against the proposal (and without such approval billing will not be allowed).
```
yes I had the same feeling, that's why I didn't translated Identity pages originally

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -139,228 +167,337 @@
+  "charts.voteTime.daysToVoteCount": "{count, plural, =0 {aucun ticket a voté} one {# ticket a voté} other {# tickets ont votés} } {days, plural, =0 {dans la même journée} one {dans la journée suivante} other {dans les # jours} } de l'achat",
   "charts.voteTime.noVotesDay": "Aucun ticket voté dans cette plage",
   "clipboard.copied": "Copié",
+  "confDialog.seed.title": "Création du portefeuille avec la graine suivante",
```
label used in : app/wallet/confirmationDialog/seed.js

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -1,45 +1,50 @@
-  "accounts.immatureRewards": "Immature Rewards",
-  "accounts.immatureStake": "Immature Stake Gen",
+  "accounts.immatureRewards": "Récompenses Immatures",
+  "accounts.immatureStake": "Récompenses d'enjeu (Stake) Immature",
```
removed as requested, no longer defined

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -139,228 +167,337 @@
   "charts.voteTime.noVotesDay": "Aucun ticket voté dans cette plage",
   "clipboard.copied": "Copié",
+  "confDialog.seed.title": "Création du portefeuille avec la graine suivante",
+  "confDialog.signTransaction.message": "Voulez-vous vraiment signer cette transaction ?",
```
label used in : app/wallet/confirmationDialog/signTx.js

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -139,228 +167,337 @@
   "clipboard.copied": "Copié",
+  "confDialog.seed.title": "Création du portefeuille avec la graine suivante",
+  "confDialog.signTransaction.message": "Voulez-vous vraiment signer cette transaction ?",
+  "confDialog.signTransaction.outputs": "Sorties allant vers un autre portefeuille:",
```
label used in : app/wallet/confirmationDialog/signTx.js

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -139,228 +167,337 @@
+  "confDialog.seed.title": "Création du portefeuille avec la graine suivante",
+  "confDialog.signTransaction.message": "Voulez-vous vraiment signer cette transaction ?",
+  "confDialog.signTransaction.outputs": "Sorties allant vers un autre portefeuille:",
+  "confDialog.signTransaction.totalOutLabel": "Total du Montant Dépensé",
```
label used in : app/wallet/confirmationDialog/signTx.js

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -139,228 +167,337 @@
+  "confDialog.signTransaction.message": "Voulez-vous vraiment signer cette transaction ?",
+  "confDialog.signTransaction.outputs": "Sorties allant vers un autre portefeuille:",
+  "confDialog.signTransaction.totalOutLabel": "Total du Montant Dépensé",
+  "confDialog.signTx.title": "Signer la Transaction",
```
label used in : app/wallet/confirmationDialog/signTx.js

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
-  "purchaseTickets.currentStakepool": "",
-  "purchaseTickets.errors.expiryRequred": "Expiration non valide",
+  "proposals.votedCheckmark.label": "Voté",
+  "purchase.splitTx": "Transaction Séparée (Split):",
```
yes scindé is more accurate

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -864,352 +1161,462 @@
+  "spv.options.disable.description": "Le Daemon Decred sera utilisé et va entièrement vérifier les blocs.  La vérification prendra plus de temps mais sera plus sécurisée.  Chaque bloc ou transaction miné pourra ensuite être considéré comme entièrement fiable.",
   "spv.options.disable.title": "Désactiver SPV",
-  "spv.options.enable.description": "SPV permet de restaurer et d’utiliser vos portefeuilles beaucoup plus rapidement. Ce gain de temps a un coût, car les blocs ne sont pas entièrement vérifiés. Cependant, même si la sécurité s’en voit diminuée, il est très peu probable que cela cause le moindre problème.",
+  "spv.options.enable.description": "SPV permet de restaurer et d’utiliser vos portefeuilles beaucoup plus rapidement. Ce gain de temps a un coût car les blocs ne sont pas entièrement vérifiés. Cependant, même si la sécurité s’en voit diminuée, il est très peu probable que cela cause le moindre problème.",
```
yes, your proposal is easier to read

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -864,352 +1161,462 @@
+  "tickets.qr.button": "QR Code des Tickets Actifs",
+  "tickets.qr.loading": "Chargement...",
+  "tickets.qr.notickets": "Aucun ticket actif dans la vue courante.",
+  "tickets.qr.tooltip": "Tickets QR",
```
Code QR des Tickets is better here

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "receive.subtitle": "Recevoir des DCR",
+  "receive.tooltipCopied": "✓  Copié dans le Presse-Papier",
+  "receive.tooltipGenerated": "+ Nouvelle Adresse Générée",
+  "receive.viewQR": "Voir le QR",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "ln.openChannel.connectTo": "Se connecter au noeud d'une contrepartie pour créer un canal et démarrer l'utilisation du Lightning Network",
+  "ln.openChannel.createChannelBt": "Créer un Canal",
+  "ln.openChannel.emptyRecentNodes": "Encore aucun Noeuds",
+  "ln.openChannel.localAmt": "Montant de Financement Total",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "ln.openChannel.createChannelBt": "Créer un Canal",
+  "ln.openChannel.emptyRecentNodes": "Encore aucun Noeuds",
+  "ln.openChannel.localAmt": "Montant de Financement Total",
+  "ln.openChannel.node": "Contrepartie (node@ip:port)",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "ln.openChannel.localAmt": "Montant de Financement Total",
+  "ln.openChannel.node": "Contrepartie (node@ip:port)",
+  "ln.openChannel.openBtn": "Ouvrir",
+  "ln.openChannel.pushAmt": "Pousser un Montant (optionnel)",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "ln.openChannel.emptyRecentNodes": "Encore aucun Noeuds",
+  "ln.openChannel.localAmt": "Montant de Financement Total",
+  "ln.openChannel.node": "Contrepartie (node@ip:port)",
+  "ln.openChannel.openBtn": "Ouvrir",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "ln.paymentsTab.hashLabel": "Hash du Paiement",
+  "ln.paymentsTab.latestPayments": "Derniers Paiements",
+  "ln.paymentsTab.lightingPayments": "Paiements Lightning",
+  "ln.paymentsTab.outstanding": "Paiements Sortants",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "ln.paymentsTab.latestPayments": "Derniers Paiements",
+  "ln.paymentsTab.lightingPayments": "Paiements Lightning",
+  "ln.paymentsTab.outstanding": "Paiements Sortants",
+  "ln.paymentsTab.outstanding.creationDate": "{creationDate, date, medium} {creationDate, time, short}",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "ln.paymentsTab.sendBtn": "Envoyer",
+  "ln.paymentsTab.sendPayment": "Envoyer un Paiement",
+  "ln.paymentsTab.sortby.tooltip": "Trier Par",
+  "ln.paymentsTag.failed": "Paiements En Échecs",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "ln.decodedPayRequestDetails.emptyFallbackAddr": "(adresse de secours vide)",
+  "ln.decodedPayRequestDetails.fallbackAddr": "Adresse de secours",
+  "ln.decodedPayRequestDetails.paymentAddr": "Adresse de paiement",
+  "ln.description.channels": "Canaux ouverts et en attente pour ce portefeuille LN",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "ln.decodedPayRequestDetails.fallbackAddr": "Adresse de secours",
+  "ln.decodedPayRequestDetails.paymentAddr": "Adresse de paiement",
+  "ln.description.channels": "Canaux ouverts et en attente pour ce portefeuille LN",
+  "ln.description.invoices": "Demandes de paiements créés par ce portefeuille LN.",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "ln.decodedPayRequestDetails.paymentAddr": "Adresse de paiement",
+  "ln.description.channels": "Canaux ouverts et en attente pour ce portefeuille LN",
+  "ln.description.invoices": "Demandes de paiements créés par ce portefeuille LN.",
+  "ln.description.network": "Informations sur l'état actuel du LN de Decred.",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "ln.description.channels": "Canaux ouverts et en attente pour ce portefeuille LN",
+  "ln.description.invoices": "Demandes de paiements créés par ce portefeuille LN.",
+  "ln.description.network": "Informations sur l'état actuel du LN de Decred.",
+  "ln.description.payments": "Paiements envoyés depuis ce portefeuille LN.",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "ln.description.payments": "Paiements envoyés depuis ce portefeuille LN.",
+  "ln.description.receive": "Créer une Demande de Paiement Lightning pour recevoir des fonds en DCR via le Lightning Network.",
+  "ln.description.send": "Utiliser le code d'une Demande de Paiement pour envoyer des fonds via le Lightning Network",
+  "ln.description.wallet": "Solde sur la blockchain et actions du portefeuille LN",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "ln.description.receive": "Créer une Demande de Paiement Lightning pour recevoir des fonds en DCR via le Lightning Network.",
+  "ln.description.send": "Utiliser le code d'une Demande de Paiement pour envoyer des fonds via le Lightning Network",
+  "ln.description.wallet": "Solde sur la blockchain et actions du portefeuille LN",
+  "ln.description.watchtowers": "Gérer les connexions aux watchtowers.",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
-  "numTicketInput.unit": "",
-  "numTicketsInput.unit": "",
-  "passphraseForm.continueBtn": "",
+  "numTicketInput.unit": "Ticket",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -371,488 +508,648 @@
+  "numTicketInput.unit": "Ticket",
+  "numTicketsInput.ticket": "ticket",
+  "numTicketsInput.tickets": "tickets",
+  "numTicketsInput.unit": "Tickets",
```
removed

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -864,352 +1161,462 @@
+  "sendtab.amountLabel": "Montant",
+  "sendtab.amountPlaceholder": "Montant",
+  "sendtab.cancelSendAllTitle": "Annuler l'envoi de tous les fonds",
+  "sendtab.deleteOutput": "Supprimer la transaction sortante",
```
I'm chosen to change it to "Supprimer la sortie" and "Ajouter une entrée" for addOutput label, it is closer to the original meaning

JaseurPattu commented on app/i18n/translations/fr.json:
```diff
@@ -864,352 +1161,462 @@
+  "trezor.recoverDevice.success": "Périphérique Trezor restauré",
+  "trezor.togglePassphraseConfirmModal.content": "La protection de la phrase secrète a été {enabled} pour {label}.{ifEnabled} Vous devez débrancher et rebrancher votre périphérique Trezor pour que ces changements soient pris en compte sur le périphérique.",
+  "trezor.togglePassphraseConfirmModal.description": "Saisir le mot de passe Trezor.",
+  "trezor.togglePassphraseConfirmModal.disable": "désactivé",
```
good catch, yes it is safer to do that

JaseurPattu commented on app/i18n/docs/fr/InfoModals/DecredConstitution.md:
```diff
@@ -0,0 +1,40 @@
+* Pour chaque bloc, 5 tickets sont appelés à voter, et sont déterminés par le [système](https://docs.decred.org/proof-of-stake/overview/) de loterie. Lorsqu'un ticket est appelé, le portefeuille nominé doit activement y répondre avec un vote. Pour qu'un bloc soit accepté par le réseau, il doit inclure les votes d'au moins 3 des 5 tickets appelés. La blockchain ne peut pas être agrandie sans la pa ...
+* Les stakeholders doivent attendre en moyenne 28 jours (8192 blocs) pour voter sur leurs tickets, pendant ce temps les DCR utilisés pour l'achat du ticket sont bloqués. L'attente peut être bien plus longue ou plus courte que la moyenne des 28 jours en raison du processus pseudo-aléatoire de sélection de ticket. Les tickets expirent après 40960 blocks (~142 jours) s'ils n'ont pas été appelés à vo ...
+* Les votes enregistrés dans la blockchain sont récompensés par 6% de la subvention de bloc, et chaque bloc peut contenir jusqu'à 5 votes, soit un total de 30% de chaque subvention de bloc.
+* Un mineur PdT/PoW reçoit 60% de chaque subvention de bloc, et à pour contrainte que la subvention augmente linéairement avec le nombre de votes PdE/PoS inclus, par exemple inclure seulement 3 votes sur 5 réduit la subvention de PoW de 60% au maximum.
```
Agree but waiting for english documentation update before doing that.

nnnko56 commented on app/i18n/docs/fr/Warnings/MixerIntroduction.md:
```diff
@@ -1,18 +1,18 @@
 "non mixé".
 
-Quand le mixer est activé, Decred va faire des transferts des fonds depuis le compte non mixé
+Quand le mixeur est activé, Des Decred seront transférés depuis le compte non mixé
```
< Des  | des  >

nnnko56 commented on app/i18n/translations/fr.json:
```diff
@@ -488,14 +487,14 @@
   "home.currentTotalSpendableBalanceLabel": "Disponible",
   "home.currentTotalUnconfirmedBalanceLabel": "Non Confirmé",
-  "home.earned": "{value} ont été gagnés en récompenses de staking",
+  "home.earned": "{value} ont été gagnés en récompenses de d'enjeu",
```
récompense {de} d'enjeu

nnnko56 commented on app/i18n/translations/fr.json:
```diff
@@ -936,24 +922,24 @@
   "myTickets.loadingMoreTicketsProgressAsc": "Bloc ascendant {block} ({blockPerc, number, percent})",
   "myTickets.loadingMoreTicketsProgressDesc": "Bloc descendant {block} ({blockPerc, number, percent})",
-  "myTickets.noloadingMoreTickets": "Chargement de plus de tickets annulé",
+  "myTickets.noloadingMoreTickets": "Chargement de plus de tickets annulés",
```
I think I was wrong on this one.
C'est le chargement qui est annulé, on ne parle pas de "tickets annulés"

nnnko56 commented on app/i18n/translations/fr.json:
```diff
@@ -1450,7 +1433,7 @@
   "transactions.title": "Transactions",
   "transactions.txtypes.tooltip": "Type de transaction",
-  "treasurySpending.description": "Toute dépense des fonds du trésor nécessite à présent un vote des stakeholders. Pour chaque clé Politeia ou transaction de type Tspend, vous pouvez voter et indiquer votre choix à propos de la dépense. Les clés peuvent être vérifiées en consultant le {link}.",
+  "treasurySpending.description": "Toute dépense des fonds du trésor nécessite à présent un vote des participants à l'enjeu. Vous pouvez participer et définir un stratégie de vote pour les dépense imputées au Trésor par une clé Politeia spécifique ou pour les transaction de type Tspend. Les clés peuvent être vérifiées en consultant le {link}.",
```
définir un[e] stratégie de vote

nnnko56 commented on app/i18n/translations/fr.json:
```diff
@@ -1581,37 +1564,37 @@
+  "tutorials.powPos.label2": "PdE (PoS)",
+  "tutorials.powPos.label3": "Hybridation PdT/PdE",
+  "tutorials.powPos.title1": "1 CPU, 1 Vote – Preuve-de-Travail",
+  "tutorials.powPos.title2": "1 Monnaie, 1 Vote – Preuve-d'Enjeu",
```
Dans ce cas-ci   peut-être que "pièce"  serait plus approprié,  aussi puisqu'on ne parle pas spécifiquement de Decred mais bien du concept général de la preuve d'enjeu,  "jeton" serait aussi acceptable je crois.

JaseurPattu replied:
Oui, pièce sonne mieux, j'avais hésité à mettre "Monnaie" ici.

nnnko56 reviewed (approved):
Great job fixing not only the 5 more "critical" elements from the first PR  but all other issues as well.

A merge is good to go!  I'm left with only 5 minor and non-urgent line comments that can be fixed now or later on

alexlyp reviewed (approved):
c'est bien
